File: Python/space.py
# Đếm khoảng trắng trong chuỗi
testcases = int(input())

while testcases > 0:
        testcases -= 1
        x = input()
        y = x.split()
        print(len(y) - 1 + int(x[0] == ' ') + int(x[-1] == ' '))

File: 11-2024/CPP/space.cpp
// Đếm khoảng trắng trong chuỗi
#include <bits/stdc++.h>
using namespace std;

string s;

int main()
{
        ios_base::sync_with_stdio(0);
        int testcases;
        cin >> testcases;
        cin.ignore();

        while (testcases--)
        {
                getline(cin, s);
                int ans = 0;
                for (int i = 0; i < s.size(); ++i)
                        if (s[i] == ' ' && (i == 0 || s[i - 1] != ' '))
                                ++ans;
                cout << ans << '\n';
        }
}